trio-oref/lib/profile/index.js

```diff
@@ -77,7 +77,7 @@ function defaults ( ) {
     , tddAdjBasal: false // Enable adjustment of basal based on the ratio of 24 h : 10 day average TDD
     , enableSMB_high_bg: false // enable SMBs when a high BG is detected, based on the high BG target (adjusted or profile)
     , enableSMB_high_bg_target: 110 // set the value enableSMB_high_bg will compare against to enable SMB. If BG > than this value, SMBs should enable.
-    , threshold_setting: 0.60 // Use a configurable threshold setting
```

Comment:
Oh my - was this always supposed to be 60 instead of 0.60?

Reply:
Yes, a typo that we didn’t introduce, but which also made no difference due to minimum limits imposed elsewhere. Please see  https://github.com/nightscout/trio-oref/pull/26 for some more info :-)